profile/nodedef/nodedefs.xml

```diff
@@ -56,6 +56,70 @@
         </sts>
     </nodeDef>
 
+    <nodeDef id="sensor01" nodeType="139" nls="139S1">

```

Comment:
Polyglot keeps track of the profile file versions in the server.json file. So when modifying these, you also need to bump the version number in there.

Reply:
There should be a "profile_version": "0.1.0", entry in the server.json, but I see that I didn't add one to this, probably because I wasn't expecting the profile files to change.  I typically start with the profile version matching the node server version and have been just incrementing it as it changes. 

I believe this allows Polyglot to recognize that there are new versions and push them automatically when the node server starts.

Reply:
I was wondering if there was an update process yet for polyglot.  I think that answers the question.

Reply:
I forgot, there's also the version.txt file as part of the profile.  That probably needs to be updated as well.  The version number in the file should match the version in the server.json.